### ISSUE TITLE
Fix nuget symbol packages for SIL.Windows.Forms.*

### ISSUE THREAD

**Issue by ermshiperete** [sillsdev/libpalaso#894]
The upload and validation of the symbol packages for SIL.Windows.Forms.dll and SIL.Windows.Forms.Keyboarding.dll fails. The reason is that both projects use ILRepack to merge with some non-strongnamed assemblies. Using ILRepack removes the necessary debug sections from the PE file.
A solution would be to create/update nuget packages for the assemblies that are not strong-named and remove ILRepack. For the Interop dlls we might want to include them in the SIL.Windows* nuget package.

Nuget packages that have to be strong-named:
- [x] DialogAdapters
- [x] MarkdownDeep.Patched
- [x] Sign the assemblies included in `lib`

**Comment by ermshiperete** [comment]
https://stackoverflow.com/a/1673465/487503 has info how to sign the assemblies in `lib`:

```
cd lib
mkdir tmp
cd tmp
ildasm ..\assembly.dll /out:assembly.il
ilasm assembly.il /dll /key=..\..\palaso.snk
```

**Comment by ermshiperete** [comment]
Fixed in #908
